.gitlab-ci.yml

```diff
@@ -26,6 +27,13 @@ dcp_wide_test_SS2:
   script:
     - python -m unittest tests.integration.test_end_to_end_dcp.TestSmartSeq2Run.test_smartseq2_run
 
+dcp_wide_test_metadata_update:
+  stage: dcp_wide_test
+  only:
+    - integration
```

Comment:
Do we not want this to run in staging or prod? (gitlab-ci-prod.yml would have to be updated)

Reply:
We can, and we definitely want this on staging. However, I'm not sure as to the nature of tests we run on prod vs the ones we run on pre-prod. In any case, we should be able to run this anywhere we deem useful. I just placed it here for now because I'm merging to integration.

Reply:
See other comment. This PR should be merged into master not integration. Updates for other environments should be made in this PR's .gitlab-ci.yml file

Reply:
I've now set the base to `master` and added `staging` to the configuration.